website/front_matter/external_considerations.qmd

```diff
@@ -0,0 +1,7 @@
+# External Considerations
+
+## Dealing With External Catastrophe
+
+Should humanity go extinct* or experience a catastrophe* that results in the collapse of complex society* for over a continuous 5 year interval, all questions in the _Genetic Evolution Tournament_ resolve as **Ambiguous**.
+
+## Dealing With Issues On Metaculus
```

Comment:
External considerations should include considerations on the absence (for whatever reason) of administrations.